Content.Server/Resist/ResistLockerSystem.cs

```diff
@@ -0,0 +1,115 @@
+using Content.Shared.Movement;
+using Robust.Shared.GameObjects;
+using Content.Server.Storage.Components;
+using Content.Server.DoAfter;
+using Robust.Shared.IoC;
+using Robust.Shared.Player;
+using Robust.Shared.Containers;
+using Content.Server.Popups;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Resist;
+
+public class ResistLockerSystem : EntitySystem
+{
+    [Dependency] private readonly DoAfterSystem _doAfterSystem = default!;
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ResistLockerComponent, RelayMovementEntityEvent>(OnRelayMovement);
+        SubscribeLocalEvent<ResistLockerComponent, ResistDoAfterComplete>(OnDoAfterComplete);
+        SubscribeLocalEvent<ResistLockerComponent, ResistDoAfterCancelled>(OnDoAfterCancelled);
+        SubscribeLocalEvent<ResistLockerComponent, EntRemovedFromContainerMessage>(OnRemovedFromContainer);
+    }
+
+    private void OnRelayMovement(EntityUid uid, ResistLockerComponent component, RelayMovementEntityEvent args)
+    {
+        TryComp(uid, out EntityStorageComponent? storageComponent);
+        if (!Resolve(uid, ref storageComponent))
+            return;
```

Comment:
This should just be:
```suggestion
        if (!TryComp(uid, out EntityStorageComponent? storageComponent))
            return;
```
`Resolve()` (defined in `EntitySystem.Resolve.cs`) is basically a TryComp that that is useful when components are optionally passed in as function arguments (which is never the case for event handlers)